frontend/src/app/testQueue/AoEForm/AoEForm.tsx

```diff
@@ -196,7 +196,13 @@ const AoEForm: React.FC<Props> = ({
   };
 
   const buttonGroup = (
-    <div className="sr-time-of-test-buttons sr-time-of-test-buttons-footer">
+    <div
+      className={classnames(
+        isModal
+          ? "aoe-modal__footer sr-time-of-test-buttons sr-time-of-test-buttons-footer"
+          : "sr-time-of-test-buttons sr-time-of-test-buttons-footer"
```

Comment:
Is there a way to do shared classes with `classnames()`?

Reply:
I believe this is what you are looking for 
```
      className={classnames("sr-time-of-test-buttons", "sr-time-of-test-buttons-footer", {"aoe-modal__footer": isModal }
```

Reply:
Perfecto, updated in 4f11e19